src/sinol_make/util.py

```diff
@@ -265,6 +268,16 @@ def get_file_md5(path):
         return hashlib.md5(f.read()).hexdigest()
 
 
+def make_version_changes():
+    """
+    Function to make changes after updating sinol-make.
+    """
+    # Move `cache` directory to `.cache` directory if the version is higher than 1.5.2.
+    if compare_versions("1.5.2", sinol_make.__version__) == -1:
+        if check_if_package() and os.path.exists("cache"):
+            shutil.move(os.path.join(os.getcwd(), "cache"), os.path.join(os.getcwd(), ".cache"))
```

Comment:
I'm not sure I understand the `if` here. When the version is strictly higher than `1.5.2`, it moves `cache` to `.cache`?

Seeing that currently sinol-make doesn't have many users, I would say it's OK to not move and just let the user have both the `cache` and `.cache` directories, just for code simplicity. I think that people that used the old sinol-make won't use the new one on the packages that they created previously.

Reply:
Sure, I deleted this part of code